ui/components/app/detected-token/detected-token-values/detected-token-values.js

```diff
@@ -46,17 +46,18 @@ const DetectedTokenValues = ({
   return (
     <Box display={DISPLAY.INLINE_FLEX} className="detected-token-values">
       <Box marginBottom={1}>
-        <Typography variant={TypographyVariant.H4}>
+        <Text variant={TextVariant.headingSm} as="h4">
```

Comment:
I think this variant matches the font size and weight better

```suggestion
        <Text variant={TextVariant.bodyLgMedium} as="h4">
```

Reply:
I was also a bit concerned about the font.
👍